### DISCUSSION TITLE
5.9.0

### DISCUSSION THREAD

**Discussion by kobylynskyi** [kobylynskyi/graphql-java-codegen#1409] [🚀 Releases]
## What's Changed
* **New option** to allow excluding fields from generated GraphQL objects: `fieldsToExcludeFromGeneration` - by @sabirove in https://github.com/kobylynskyi/graphql-java-codegen/pull/1299
* **New option** to disable GraphQL schema token limit validation: `skipSchemaSizeLimit` by @yholkamp in https://github.com/kobylynskyi/graphql-java-codegen/pull/1384
* **New option** to render Kotlin functions with `suspend` modifier: `generateApisWithSuspendFunctions` - by @thevietto in https://github.com/kobylynskyi/graphql-java-codegen/pull/1377
* Support `all$` method for union and interface projections - by @sdaurea in https://github.com/kobylynskyi/graphql-java-codegen/pull/1376
* Fix enum generation when `customTypesMapping` and `directiveAnnotationsMapping` are used together - by @jxnu-liguobin in https://github.com/kobylynskyi/graphql-java-codegen/pull/1298
* Fix mapping config for `generateModelsWithPublicFields` - by @ssternal in https://github.com/kobylynskyi/graphql-java-codegen/pull/1310
* Fix value sanitization for immutable collections - by @kobylynskyi in https://github.com/kobylynskyi/graphql-java-codegen/pull/1408

(See [docs/codegen-options.md](docs/codegen-options.md) for more details on all the available codegen options.

## New Contributors
* @sabirove made their first contribution in https://github.com/kobylynskyi/graphql-java-codegen/pull/1299
* @ssternal made their first contribution in https://github.com/kobylynskyi/graphql-java-codegen/pull/1310
* @thevietto made their first contribution in https://github.com/kobylynskyi/graphql-java-codegen/pull/1377
* @sdaurea made their first contribution in https://github.com/kobylynskyi/graphql-java-codegen/pull/1376
* @yholkamp made their first contribution in https://github.com/kobylynskyi/graphql-java-codegen/pull/1384

**Full Changelog**: https://github.com/kobylynskyi/graphql-java-codegen/compare/v5.8.0...v5.9.0

<hr /><em>This discussion was created from the release <a href='https://github.com/kobylynskyi/graphql-java-codegen/releases/tag/v5.9.0'>5.9.0</a>.</em>

**Comment by kobylynskyi** [comment]
@jxnu-liguobin please proceed with SBT plugin release. Thanks!

**Reply by jxnu-liguobin** [reply]
I have added auto-release support for sbt, it still needs to be configured with my key (or move to kobylynskyi from next release?)
